GNUmakefile

```diff
@@ -24,12 +27,31 @@ egg:
 	# setup.py will call 'make build' before creating the egg
 	python setup.py bdist_egg
 
-build:
-	cd $(PYWBEM_DIR) ; \
-		PYTHONPATH="$(PYTHONPATH):$(LIB_DIR)" \
-		$(PYTHON) setup.py install \
-		--install-lib="$(LIB_DIR)" \
-		--install-scripts="$(BIN_DIR)"
+build: typing m2crypto ply pywbemz $(LIB_DIR) $(BIN_DIR)
+	@echo
+
+$(LIB_DIR):
+	mkdir -p $(LIB_DIR)
+	PYTHONPATH="$(PYTHONPATH):$(LIB_DIR)"
+
+$(BIN_DIR):
+	mkdir -p $(BIN_DIR)
+
+ply: $(LIB_DIR) $(BIN_DIR)
+	PYTHONPATH="$(PYTHONPATH):$(LIB_DIR)" \
+	easy_install --no-deps --install-dir="$(LIB_DIR)" --script-dir="$(BIN_DIR)" $(PLY_TAR)
+
+typing: $(LIB_DIR) $(BIN_DIR)
+	PYTHONPATH="$(PYTHONPATH):$(LIB_DIR)" \
+	easy_install --no-deps --install-dir="$(LIB_DIR)" --script-dir="$(BIN_DIR)" $(TYPING_TAR)
+
+m2crypto: typing
+	PYTHONPATH="$(PYTHONPATH):$(LIB_DIR)" \
+	easy_install --no-deps --install-dir="$(LIB_DIR)" --script-dir="$(BIN_DIR)" $(M2CRYPTO_TAR)
```

Comment:
You're still installing all of these dependencies into ZenPacks/zenoss/WBEM/lib/ which addLocalLibPath() is still adding to sys.path as soon as any Zenoss process starts (see `import ZenPacks.zenoss.WBEM.patches` in `__init__.py`).

Reply:
I've tried many combinations of loading the modules through dependency.py.  The end result was a non-functional zenpack; this has been the only working solution so far.